spec/new_integration/rack_app/rack_app_spec.rb

```diff
@@ -7,6 +7,30 @@
 
   let(:app) { Hanami.rack_app }
 
+  specify "has rack monitor preconfigured" do
```

Comment:
Would it be worth expanding on this description a little bit? Right now it's a bit of a confusing leap from "has rack monitor preconfigured" to "there are logs from web requests". Is this test actually more about asserting the web requests are logged, with rack monitor being more of an implementation detail?

Reply:
Good point. I'll make it more clear and explicit.